### PR TITLE
[release-1.30] Code Thaw and milestone applier for 1.31

### PR DESCRIPTION
**Pull request by marosset**
- Lifts code freeze for k/k
- Updates milestone applier for some k/* branches to v1.31

/sig release
/assign @katcosgrove

### CODE REVIEW

File: config/prow/plugins.yaml [hunk 1]
```diff
@@ -459,28 +459,28 @@ slack:
 
 milestone_applier:
   kubernetes/enhancements:
-    master: v1.30
+    master: v1.31
   kubernetes/kubernetes:
-    master: v1.30
+    master: v1.131
```
Comment (BenTheElder) [inline config/prow/plugins.yaml:464]:
```suggestion
    master: v1.31
```
?

**Comment by katcosgrove** [comment]
/hold cancel

WOOOOOOOOO

**Pull request merged**